openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v160/MigrationUtil.java

```diff
@@ -138,4 +139,30 @@ private static void migrateConnectionTypeInApiServiceEntity(Handle handle, boole
       LOG.error("Error updating", e);
     }
   }
+
+  public static void addDisplayNameToCustomProperty(Handle handle, boolean postgresql) {
+    String query;
+    if (postgresql) {
+      query =
+          "UPDATE field_relationship "
+              + "SET json = jsonb_set(json, '{displayName}', jsonb_path_query_first(json, '$.name'), false) "
```

Comment:
is `$.name` an array that we use `jsonb_path_query_first` to access the element?

Reply:
its of type text, updated the query . Thanks 👍